### DISCUSSION TITLE
Use unicode emojis instead of flags

### DISCUSSION THREAD

**Discussion by muscal** [goveo/react-international-phone#163] [Ideas]
Hey there,

There is concern over opening up the CDN to retrieve the flag images. Can you replace the flags with unicode emojis so we don't have to use images.

This also seems like it would be a lot more scalable w/ less maintenance. 

Thanks!

**Comment by ybrusentsov** [comment]
Hey, unfortunately, unicode flags don't display properly in all browsers, so I can't set them as the default.

**Comment by ybrusentsov** [comment]
@muscal Currently, you can avoid CDN usage by passing svg files for all countries to the `flags` prop.
https://react-international-phone.vercel.app/docs/Usage/CustomCountryFlags/
